src/libraries/System.ComponentModel.TypeConverter/tests/CultureInfoConverterTests.cs

```diff
@@ -152,6 +153,15 @@ public void GetCultureName_Overriden_ConversionsReturnsExpected()
             Assert.Equal("Fixed", converter.ConvertTo(new CultureInfo("en-US"), typeof(string)));
         }
 
+        [Fact]
+        public void CultureInfoConverterForDefaultValue()
+        {
+            using (new ThreadCultureChange(null, CultureInfo.InvariantCulture))
+            {
+                Assert.Equal("", ((CultureInfo)TypeDescriptor.GetConverter(typeof(System.Globalization.CultureInfo)).ConvertFrom(null, null, "(Default)")).Name);
```

Comment:
Don't we need to run this test with `UseSystemResourceKeys=true` in order to test the bug?

Reply:
I have manually run the test with and without `UseSystemResourceKeys`. As @jkotas indicated, such mode should be tested for all libraries and not just this one only. I'll open infrastructure issue to do that.

Reply:
I'm not sure when we would enable a full test run with this setting. It doesn't seem like something that will happen anytime soon.

For now, we can run this one test with both `UseSystemResourceKeys=true or false` by doing something like the following code, only with the UseSystemResourceKeys switch:

https://github.com/dotnet/runtime/blob/66bd8b193bdaa96e26308e6c056f0b27434428cd/src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestNotSupported.cs#L18-L30

Reply:
ok, will do it.